product_ext_sst/views/product_template_views.xml

```diff
@@ -41,9 +41,8 @@
                 <field name="list_price" optional="show" />
                 <field name="carrier_id" optional="show" />
                 <field name="carrier_size_id" optional="show" />
-                <field name="selectable_delivery_city_ids" invisible="1" />
-                <field name="delivery_state_id" optional="show" />
-                <field name="delivery_city_id" optional="show" />
+                <field name="deliver_prefecture" optional="show" />
```

Comment:
Not the main point but the field name should have been `delivery_prefecture` to keep consistency with other fields.